spec/GraphQLOverHTTP.md

```diff
@@ -112,13 +112,29 @@ http://example.com/product/graphql
 
 # Serialization Format
 
-Similar to the GraphQL specification this specification does not require a specific serialization format.
+The GraphQL specification allows for many [serialization formats to be implemented](https://spec.graphql.org/June2018/#sec-Serialization-Format). Servers and clients over HTTP MUST support JSON and MAY support other, additional serialization formats.
+
 For consistency and ease of notation, examples of the response are given in JSON throughout the spec.
 
+## Content Types
+
+The following are the officially recognized GraphQL content types to designate encoding JSON over HTTP.
+
+| Name | Description |
+| `application/graphql+json` | Required
+| `application/json` | To support legacy clients
+
+A server MUST support requests from clients with HTTP header `Content-Type: application/graphql+json` indicating that the body of the request is a JSON document with a GraphQL request. A server must indicate the content type of the response with a `Content-Type` header. This default value should be `Content-Type: application/graphql+json` indicating that the response is a valid JSON document containing at least one of the `data` or `errors` as a top-level attribute.
```

Comment:
The last part could refer to the relevant spec instead of reiterating it in a simplified form.